pkg/version/version_test.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+import (
+	"errors"
+	"testing"
+)
+
+var (
+	a = &Build{
+		Version:     "v1.2.3",
+		GitCommit:   "aaaaaaa",
+		BuildDate:   "2009/11/10@23:00:00",
+		BuildNumber: "10",
```

Comment:
Think I saw @emlin say something about this, but is there a reason BuildNumber is a string instead of an integer? Makes me nervous with those `<`s; I think that's what Emma said too, though.